src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/NamingUtilTests.cs

```diff
@@ -0,0 +1,86 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Content.Model.NamingUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model.Tests
+{
+    public class NamingUtilTests
```

Comment:
You don't test any TimeIdentifiers here that are Quarters, groups of Quarters, Terms, groups of Terms, Months, EndOfMarch etc here.

Also, possible overlap with the tests in TimePeriodLabelFormatterTests. See my previous comment about the overlap between the NamingUtil and TimePeriodLabelFormatter.

Reply:
year stuff is now all covered with the timeformatter stuff